---
id: 484
title: CV and contact
date: 2015-12-05T03:02:17+00:00
author: bradford.condon@gmail.com
layout: page
permalink: cv.html
---

<p style="text-align: center;">
  Bradford Condon, PhD
</p>

## Education
- August 2013, PhD in Plant Pathology & Plant-Microbe Interactions, Cornell University, Ithaca, NY
- May 2007, BA in Biology and Religion, Oberlin College, Oberlin, OH

# Bradford the Web Developer and Technologist (2019-present)

## Employment

* October 2022 to present: Director of Technology, [CABEM Technologies](https://www.cabem.com/)
> Guide technology decisions across all client projects in CABEM.  Piloted automated testing, documentation, and publish harnesses (PHPUnit, Jest, Jest-axe, react-styleguidist, doxygen, selenium, gitlab's npm/composer package registry). Gather requirements from clients, write and manage jira tickets to lead launching two custom, enterprise level applications, while implementing the most critical frontend, backend, and infrastructure components.


* April 2019 to present: Full stack Software Developer, [CABEM Technologies](https://www.cabem.com/)

> Develop and implement OAS 3.0 compliant API using PHP Slim and custom frameworks.  Design and build UIs with React. Manage support and development workflow with gitlab and Jira. Pioneer transition from manual to automated deployments using AWS CodeBuild, CodeDeploy, via CodePipelines.

## Technologies

- Application builds with AWS: lambda, Step functions, CodeBuild, CodePipeline, CodeDeploy.
- Networking with AWS: Route 53, ALB, ELB, EC2, ASG.
- Backend: PHP, PHPSlim, Laravel subcomponents, Apache, Symfony, Python.
- Frontend: React, Redux, Axios, CSS, SCSS, Bootstrap.

See academic section below for archived technology expertise and language (such as R, Perl, and Ruby).

# Bradford the Scientist and Academic (2003-2019)

## Employment

* September 2018 to April 2019: IT Specialist III, University of Tennessee

> Tripal core developer.  Collaborative Tripal developer for the Hardwood Genomics Web and the USDA-NAL I5K project.

* February 2017 to September 2018: Postdoctoral associate, Department of Entomology and Plant Pathology, University of Tennessee, Knoxville.

>I develop the Hardwood Genomics Web (HWG), a web site that houses genetic and genomic data from a variety of hardwood tree species.  I contribute to Tripal, an open source genomic database and website framework. I also authored the mobile app TreeSnap, which allows citizens to partner with scientists to track and study trees.  I serve as the biological point person for undergraduate and graduate lab programmers to help them understand the underlying database model.

* January 2016 to January 2017: Postdoctoral associate, Department of Plant Pathology, University of Kentucky.

>Performed bioinformatic analyses for whole-genome alignment and phylogenetics of the rice blast pathogen _magnaporthe oryzae_.  Developed and implemented a mentoring program in bioinformatics for undergraduate biologists, UK-BLAST.

* June 2014 to December 2015: Postdoctoral associate, Department of Ecology, Evolution, and Behavior, University of Minnesota.
>Planned and implemented a large-scale metagenomic study of bacterial and fungal endophytes for over 1,000 native prairie grass leaves.  Designed a metagenomic sequencing and analysis pipeline.

* August 2007 to June 2014: Graduate Student and Post-Doctoral fellow, Department of Plant Pathology & Plant-Microbe Interactions, Cornell University, advised by Dr. [B Gillian Turgeon](http://www.plantpath.cornell.edu/labs/turgeon/index.html)


>Performed genetic and genomic studies using the model fungal phytopathogen Cochliobolus heterostrophus.  Acted as lab co-manager in charge of ordering and hiring/training/managing undergraduate students.



## Accomplishments

* [Tripal](tripal.info) project manager.
   - Established testing protocols and continuous integration testing.
   - Formed a governance structure with contribution guidelines and requirements.
* Upgraded [HardwoodGenomics](https://hardwoodgenomics.org/) to Tripal 3, performed bioinformatic analyses and uploaded new data to site.
* Co-developer of [TreeSnap](https://treesnap.org/), mobile citizen science app.



## Selected Software

#### Tripal

* [Tripal Headquarters](https://github.com/statonlab/tripal_hq) [![DOI](https://zenodo.org/badge/152087832.svg)](https://zenodo.org/badge/latestdoi/152087832)
  - Provide an admin layer between Chado and Tripal
  - Chado-specific permissions, allowing administrators to create community-specific roles within the site
* [Tripal Analysis Expression Module](https://github.com/tripal/tripal_analysis_expression)
  - For loading and display of RNASeq data, from biological source material to gene expression data  
  - DOI: 10.5281/zenodo.1115662
* [Tripal ElasticSearch Module](https://github.com/tripal/tripal_elasticsearch)
  - For installing and managing sitewide search, customized search forms, and cross-Tripal-site queries via ElasticSearch
  - DOI: 10.5281/zenodo.1193925
* [Tripal Fields Generator](https://github.com/statonlab/fields_generator)
  - Developer CLI tool to automate Tripal field creation for Tripal v3 development
  - DOI: 10.5281/zenodo.1200662
* [Tripal TestSuite](https://github.com/statonlab/TripalTestSuite)
  - A composer package that handles common test practices such as bootstrapping Drupal before running the tests, creating test file, and creating and managing database seeders.
  - DOI: 10.5281/zenodo.1209904
* [Tripal Alchemist](https://github.com/statonlab/tripal_alchemist)
  - Enables bulk transformation of Tripal entities from one type to another
  - DOI: 10.5281/zenodo.1187120



#### TreeSnap

Lead developer for TreeSnap, a citizen science mobile app.  Find out more: https://treesnap.org/

Code:

* [Mobile](https://github.com/statonlab/Treesnap-mobile)
* [Web](https://github.com/statonlab/Treesnap-website)


## Skills

#### Bioinformatics and programming

  * Proficient with UNIX/LINUX systems, command-line programming, querying and managing big data/databases
  * Developer for cross-platform (iOS and Android) apps in React-Native, and genomics websites using Tripal and Drupal.
  * Experience designing experiments and analyzing data for a wide variety of bioinformatic projects, including
      * Metagenomics
      * RNA-seq and transcriptomics
      * Genome assembly and annotation
      * Phylogenetics

  * Programming languages: **R, Perl, javascript, PHP, CSS,SASS, Ruby, Python, SQL**
  * Frameworks: **Laravel, Drupal, Wordpress, React, React Native**

#### Molecular biology

  * Fungal and bacterial culture, transformation, and phenotyping
  * Molecular bio bench skills include DNA extraction, PCR, southern/northern blotting, light and confocal microscopy
  * Next-gen (illumina) library preparation
  * Greenhouse management

#### Communication and mentorship

  * Formed UK-BLAST, the undergraduate bioinformatics mentorship group at the University of Kentucky
      * Leading and designing workshops in bioinformatics
      * Co-mentoring and advising students on their bioinformatics projects
  * Strong track record of supervising undergraduates in the laboratory and integrating their work into complex workflows
  * Co-PI awardee for bringing [Expanding Your Horizons](http://www.eyhn.org/), a STEM conference for young girls, to the University of Kentucky
  * Community outreach and education on fungi includes presentations at Floracliff Nature Sanctuary and the Kentucky Food Retail Safety conference

#### Awards and honors

* 2006 Oberlin College Norman H Wright Prize for undergraduate research
* 2007-2008 Presidential Life Science Fellow
* 2009 NSF graduate research fellowship honorable mention
* 2010 NSF EAPSI fellow (New Zealand)
* 2011 Plant Sciences outstanding TA award
* 2012 McClintock Outstanding Plant Sciences Graduate Student Award
* 2011-2012 Center For Teaching Excellence Graduate Teaching Associate Fellow
* 2012-2013 Center For Teaching Excellence Graduate Teaching Associate Fellow
* 2017 NSF EPSCoR grant, [Expanding Your Horizons](http://www.eyhn.org/)
* 2017 [UKY SOPS](http://sops.as.uky.edu/?q=content/symposia) research symposium postdoc poster award (second place)

#### Teaching and leadership

##### Data and Software Carpentry Certified Instructor 2017
Trained and certified as a [Data Carpentry](http://www.datacarpentry.org/) and [Software Carpentry](https://software-carpentry.org/) Instructor.

[Blog posts regarding SWC/DC](/tag/software-carpentry/).

**Hosted Workshops**

* [December 18th-19th 2017, Knoxville, TN](https://statonlab.github.io/2017-12-18-carpentry-workshop/)
  - Topics covered: Unix, Git, and R

##### KBRIN Next Generation Sequencing and Data Analysis Workshop, *University of Kentucky* 2017

Instructor and web developer.  Implemented and maintained the (Drupal-based workshop website)[https://ngs.csr.uky.edu/] and REDCap registration/survey system.  [Poster publication link](https://link.springer.com/article/10.1186/s12859-017-1781-y).

##### UK-BLAST: *University of Kentucky* 2016

UK-BLAST is an extracurricular support group for undergraduates performing genomics research at the University of Kentucky.  In addition to advising students on their projects, I designed and implemented workshops on topics in genomics.


##### Departmental teaching: _Cornell University_ 2008-2013

* Spring 2008: PLPA 2010 TA: Magical Mushrooms & Mischievous Molds.
* Spring 2010: PLPA 2015 TA: Mushrooms, Molds and Molecules.
* Spring 2011 PLPA 2015 TA: Mushrooms, Molds, and Molecules.
* Spring 2012 PLPA 2015 TA:  Mushrooms, Molds, and Molecules.
* Spring 2013 PLPA 2015 TA:  Mushrooms, Molds, and Molecules.

##### [Center for Teaching Excellence](http://www.cte.cornell.edu/): _Cornell University_ 2011-2013

_workshops (1 hour) developed & implemented:_

* Classical teaching models and the psychology of learning
* What do I want my students to learn? Identifying learning outcomes as the basis for instruction and assessment
* Using library resources to enhance student’s research skills
* Creating effective questions for iClickers and online quizzes
* Establishing a professional electronic online identity

_Full day workshops developed & implemented:_

* 2012 Chemistry TA training instructor
* 2012 Teaching Excellence Practicum Instructor, CTE summer Institute

#### Presentations

##### Organizer

* [Plant and Animal Genomes Conference](http://www.intlpag.org/) Tripal Session Organizer.  San Diego CA (2019)

##### Workshops
* [Floracliff mushroom identification workshop](http://www.bradfordcondon.com/2017/04/17/floraCliffUpdate/). Lexington, KY (2016)
* [AgBioData metadata committee](https://www.agbiodata.org/working_groups/metadata_persistence_wg). Salt Lake City, UT (2017)
* [GMOD Hackathon](http://www.bradfordcondon.com/2018/01/12/gmod_day_1_2018/), [Plant and Animal Genomes Conference](http://www.intlpag.org/). San Diego CA (2018)
* [Tripal 2018 Hackathon](http://www.bradfordcondon.com/2018/01/17/tripal_at_pag_2018/) Instructor, [Plant and Animal Genomes Conference](http://www.intlpag.org/). San Diego CA (2018)
* Tripal 2019 Hackathon group leader, [Plant and Animal Genomes Conference](http://www.intlpag.org/). San Diego CA (2019)

##### Talks
* Joint Genome Institute Dothideomycete jamboree. Walnut Creek, CA (2008)
* Fungal Genetics Conference, secondary metabolism session. Asilomar, CA (2013)
* Kentucky Food Retail Safety Conference, invited speaker on wild mushroom safety. Frankfort, KY (2016)
  - [Slides](https://www.slideshare.net/BradfordCondon1/ky-retail-foodseminar)
* TreeSnap: A citizen Science Project and Mobile App to Tag Trees for Resistance.  Tree Resistance Conference, OH (2018)
  - [Slides](https://www.slideshare.net/BradfordCondon1/treesnap-tree-resistance-conference-2018)
* Tripal v3, the Collaborative Online Database Platform Supporting an International Community of Plant Genome Databases.  [Plant and Animal Genome Conference (PAG)](http://www.intlpag.org/2019/), San Diego, CA (2019)
  - [Slides](https://www.slideshare.net/BradfordCondon1/tripal-v3-the-collaborative-online-database-platform-supporting-an-international-community-of-plant-genome-databases)

##### Posters

* American Society of Plant Biologists, Boston, MA (2006)
* Fungal Genetics Conference, Asilomar, CA (2009)
* Molecular Plant Microbe Biology, Quebec City, QC (2009)
* Fungal Genetics Conference, Asilomar, CA (2011)
* [University of Kentucky Society of Postdoctoral Scholars Research Symposium](http://sops.as.uky.edu/), Lexington, KY (2017)
* [Oak Symposium](http://www.outreach.utk.edu/Conferences/Sustaining%20Oak%20Forests/announcement.html), Knoxville TN (2017)
* Tree Resistance Conference, OH (2018)


## Publications

Crocker, E, **Condon, B,** Almsaeed, A, Jarret, B, Nelson, C D, Abbot, A, Main, D, Staton, M (2019) TreeSnap: A citizen science app connecting tree enthusiasts and forest scientists  Plants, People, Planet https://doi.org/10.1002/ppp3.41

Spoor, S, Cheng, C, Sanderson, L-A,  **Condon, B**, et al. (2019) Tripal v3: an ontology-based toolkit for construction of FAIR biological community databases. Database, baz077, https://doi.org/10.1093/database/baz077

**Condon, B**, Almsaeed, A, West, J, Chen, M, Staton, M (2018).  Tripal Developer Toolkit.  Database, bay099 DOI: https://doi.org/10.1093/database/bay099

Lumibao, C, **Condon, B**, Kinkel , L, May, G, Seabloom, E, Borer, E Site‐specific responses of foliar fungal microbiomes to nutrient addition and herbivory at different spatial scales.  Ecology and Evolution, October 2019. https://doi.org/10.1002/ece3.5711


Seabloom, E, **Condon, B**, Kinkel, L, Komatsu, K, Lumibao, C, May, G, McCulley, R, Borer, E.(2019) Effects of nutrient supply, herbivory, and host community on fungal endophyte diversity.  Ecology,  . https://doi.org/10.1002/ecy.2758


**Condon, B.**, Elliott, C, Gonzalez, J, Yun, S, Akagi, Y, Wiesner-Hanks, T, Kodama, M, Turgeon, G (2018).  Clues to an evolutionary mystery: the genes for T-toxin, enabler of the devastating 1970 Southern Corn Leaf Blight epidemic, are present in ancestral species, suggesting an ancient origin.  _MPMI_ Molecular Plant-Microbe Interactions 2018 31:11, 1154-1165.  DOI: https://doi.org/10.1094/MPMI-03-18-0070-R

Lisa Harper, Jacqueline Campbell, Ethalinda K S Cannon, Sook Jung, Monica Poelchau, Ramona Walls, Carson Andorf, Elizabeth Arnaud, Tanya Z Berardini, Clayton Birkett, Steve Cannon, James Carson, **Bradford Condon**, Laurel Cooper, Nathan Dunn, Christine G Elsik, Andrew Farmer, Stephen P Ficklin, David Grant, Emily Grau, Nic Herndon, Zhi-Liang Hu, Jodi Humann, Pankaj Jaiswal, Clement Jonquet, Marie-Angélique Laporte, Pierre Larmande, Gerard Lazo, Fiona McCarthy, Naama Menda, Christopher J Mungall, Monica C Munoz-Torres, Sushma Naithani, Rex Nelson, Daureen Nesdill, Carissa Park, James Reecy, Leonore Reiser, Lacey-Anne Sanderson, Taner Z Sen, Margaret Staton, Sabarinath Subramaniam, Marcela Karey Tello-Ruiz, Victor Unda, Deepak Unni, Liya Wang, Doreen Ware, Jill Wegrzyn, Jason Williams, Margaret Woodhouse, Jing Yu, Doreen Main (2018) AgBioData consortium recommendations for sustainable genomics and genetics databases for agriculture, _Database_, Volume 2018, bay088, DOI: https://doi.org/10.1093/database/bay088


Gladieux, P., **Condon, B. J.** (co-first author), Ravel, S., Soanes, D., Maciel, J. L. N., Nhani, A., Chen, L., Terauchi, R., Lebrun, M.-H., Tharreau, D., Mitchell, T., Pedley, K. F., Valent, B., Talbot, N. J., Farman, M., and Fournier, E. (2018). Gene Flow between Divergent Cereal- and Grass-Specific Lineages of the Rice Blast Fungus _Magnaporthe oryzae_. J.W. Taylor, ed. _MBio_. 9:e01219–17.  DOI: 10.1128/mBio.01219-17


Mohd Zainudin, N. A. I., **Condon, B. J.,** De Bruyne, L., Poucke, C. V., Bi, Q., Li, W., et al. (2015). Virulence, host selective toxin production, and development of three Cochliobolus phytopathogens lacking the Sfp-type 4&#8242;-phosphopantetheinyl transferase Ppt1. _MPMI_, 28:10, 1130-1141. DOI: 10.1094/MPMI-03-15-0068-R

Santana, M. F., Silva, J. C. F., Mizubuti, E. S. G., Araújo, E. F., **Condon, B. J.,** Turgeon, B. G., & Queiroz, M. V. (2014). Characterization and potential evolutionary impact of transposable elements in the genome of _Cochliobolus_ _heterostrophus_. _BMC Genomics_, _15_(1), 536. doi:10.1186/1471-2164-15-536

**Condon, B. J.,** Wu, D., Kraševec, N., Horwitz, B. A., & Turgeon, B. G. (2014). Comparative Genomics of _Cochliobolus_ Phytopathogens. In R. A. Dean, A. Lichens-Park, & C. Kole, _Genomics of Plant-Associated Fungi: Monocot Pathogens_ (pp. 41–67). Berlin, Heidelberg: Springer Berlin Heidelberg. doi:10.1007/978-3-662-44053-7_2

**Condon, B. J.,** Oide, S., Gibson, D. M., Krasnoff, S. B., & Turgeon, B. G. (2014). Reductive iron assimilation and intracellular siderophores assist extracellular siderophore-driven iron homeostasis and virulence. _Molecular Plant-Microbe Interactions : MPMI_, _27_(8), 793–808. doi:10.1094/MPMI-11-13-0328-R

Horwitz, B. A., **Condon, B.**, & Turgeon, B. G. (2013). Cochliobolus heterostrophus: A Dothideomycete Pathogen of Maize. In _Genomics of Soil-and Plant-associated Fungi_ (Vol. 36, pp. 213–228). Berlin, Heidelberg: Springer Berlin Heidelberg. doi:10.1007/978-3-642-39339-6_9

Zhang, N., MohdZainudin, N. A. I., Scher, K., **Condon, B. J.,** Horwitz, B. A., & Turgeon, B. G. (2013). Iron, oxidative stress, and virulence: roles of iron-sensitive transcription factor Sre1 and the redox sensor ChAp1 in the maize pathogen _Cochliobolus heterostrophus_. _Molecular Plant-Microbe Interactions : MPMI_, _26_(12), 1473–1485. doi:10.1094/MPMI-02-13-0055-R

**Condon, B. J.,** Leng, Y., Wu, D., Bushley, K. E., Ohm, R. A., Otillar, R., et al. (2013). Comparative genome structure, secondary metabolite, and effector coding capacity across _Cochliobolus_ pathogens. _PLoS Genetics_, _9_(1), e1003233. doi:10.1371/journal.pgen.1003233

Xue, C., Wu, D., **Condon, B.**, Bi, Q., Wang, W., & Turgeon, B. G. (2013). Efficient gene knockout in the maize pathogen _Setosphaeria turcica_ using _Agrobacterium tumefaciens_-mediated transformation. _Phytopathology_, _103_(6), 641–647. doi:10.1094/PHYTO-08-12-0199-R

Ohm, R. A., Feau, N., Henrissat, B., Schoch, C. L., Horwitz, B. A., Barry, K. W., **Condon, B. J.,** et al. (2012). Diverse lifestyles and strategies of plant pathogenesis encoded in the genomes of eighteen Dothideomycetes fungi. _PLoS Pathogens_, _8_(12), e1003037. doi:10.1371/journal.ppat.1003037

Turgeon, B. G., **Condon, B. J**., Liu, J., & Zhang, N. (2010). Protoplast transformation of filamentous fungi. _Methods in Molecular Biology (Clifton, N.J.)_, _638_(Chapter 1), 3–19. doi:10.1007/978-1-60761-611-5_1

Almeida, N. F., Yan, S., Lindeberg, M., Studholme, D. J., Schneider, D. J., **Condon, B. J.,** et al. (2009). A draft genome sequence of _Pseudomonas syringae_ pv. tomato T1 reveals a type III effector repertoire significantly divergent from that _of Pseudomonas syringae_ pv. tomato DC3000. _Molecular Plant-Microbe Interactions : MPMI_, _22_(1), 52–62. doi:10.1094/MPMI-22-1-0052

# Bradford The Child (The 80's-2003)

- Born and raised in Connecticut